compiler/src/pipeline.rs

```diff
@@ -158,7 +160,7 @@ where
             log_level: Level::Debug,
             name: None,
             force_overwrite: false,
-            tmp_dir: None,
+            _tmp_dir: None,
```

Comment:
Why did you add the underscore?

Reply:
Because it is no longer used (and the linter complains about it unless you start with underscore). As explained in the docstring of `_tmp_dir`, it is only kept in Pipeline so that it has an owner. The directory will live for as long as the pipeline does.